### PR TITLE
Update for public use.

### PR DESCRIPTION
**Pull request by jeanconn**
## Description

This PR:

- Removes references to text files no longer supported.
- Updates the text to explain what the agasc package is.

### Unit tests

Docs only update - tests not run

### CODE REVIEW

**Comment by jeanconn** [comment]
Though if we're going to include the catalog-description.rst as the public docs for AGASC 1.8, it probably needs a link within it to the github docs to explain what an "agasc package query" is.

File: docs/catalog-description.rst [hunk 1]
```diff
@@ -69,21 +69,30 @@ Version 1.8 includes improved star positions and proper motions from Gaia
...
+directly with that Python package.
+
+
 Organization of the FITS Data Files
 ===================================
 
 The top directory includes the files::
 
     ----------------------------------------------------------------
-    readme.txt    - Introduction. ASCII
```
Comment (jeanconn) [inline docs/catalog-description.rst:78]: Question here if we should add catalog-description.rst to the file list or not bother.

Comment (javierggt) [reply docs/catalog-description.rst:78]: I was thinking this file would be renamed readme.txt when placing it with the fits file distribution.

The `readme.txt` could be replace by `this file`

Comment (javierggt) [reply docs/catalog-description.rst:78]: either way, just removing the line is ok for me.

Comment (jeanconn) [reply docs/catalog-description.rst:78]: OK, if you were thinking that it was fine to drop the ".rst" extension and just call catalog-description.rst the new readme.txt that's fine too.

Comment (jeanconn) [reply docs/catalog-description.rst:78]: Done.

**Comment by jeanconn** [comment]
I'm planning to ignore unrelated ruff issues.

**Review by javierggt — APPROVED** [review]

**Pull request merged**